### ISSUE TITLE
A few problems

### ISSUE THREAD

**Issue by lukazsj** [Radiergummi/kiosk#5]
I've tried to install kiosk, but some functions don't really run.

There's no read function on BooksController.
Authors and Publishers aren't even listed.

is there a functional branch?

**Comment by Radiergummi** [comment]
Yup, its broken at the moment. The last mostly complete version was at [`1d7f2bc`](https://github.com/Radiergummi/kiosk/tree/1d7f2bcf87059ccb4c186fb767b2ba0707836a40), before I started migrating the application to Laravel.  
I never anticipated any public reaction to the project and I would love to get it up and running again, but I'm pretty low on free time currently. 

I'm trying to make some and get back to this, though!
